yamlExamples/1/pipeline

```diff
@@ -0,0 +1,46 @@
+pipeline{
+    agent{
+        node{
+            label 'my_lable'
+            customWorkspace 'path_to_workspace'
+        }
+    }
+
+    tools{
+        tool_1 'tool_1_name'
+        tool_2 'tool_2_name'
+    }
+
+    environment{
+        variable_1 = 'value_1'
+        variable_2 = 'value_2'
+    }
+
+    stages{
+        stage('Build'){
+            steps{
+                withCredentials([[$class: 'usernamePassword', credentialsId: 'usernamePasswordId',
+                usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD']]) {
+                    withCredentials([file(credentialsId: 'fileCredentialId', variable: 'FILE')]) {
+                        sh 'path_1.sh'  // or bat
```

Comment:
"// or bat"  -> I'd just create another example with bat as the command.  Make it a 1-to-1 translation of your yaml file.